packages/office-ui-fabric-react/src/components/Button/ButtonCore/ButtonCoreVars.scss

```diff
@@ -1,3 +1,16 @@
 $button-core-default-height: 32px;
 $button-core-default-minWidth: 80px;
 $button-core-default-padding: 0 16px;
+
+@mixin core-button-disabled {
```

Comment:
Mixins are global, so it's best to prefix with 'ms' and the component name. Something like `ms-Button-core-disabled()`